x-pack/plugins/upgrade_assistant/public/components/tabs/checkup/deprecations/reindex/flyout/checklist_step.tsx

```diff
@@ -25,15 +27,40 @@ import { ReindexProgress } from './progress';
 const buttonLabel = (status?: ReindexStatus) => {
   switch (status) {
     case ReindexStatus.failed:
-      return 'Try again';
+      return (
+        <FormattedMessage
+          id="xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.reindexButton.tryAgainLabel"
+          defaultMessage="Try again"
+        />
+      );
     case ReindexStatus.inProgress:
-      return 'Reindexing…';
+      return (
+        <FormattedMessage
+          id="xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.reindexButton.inProgressLabel"
+          defaultMessage="Reindexing…"
+        />
+      );
     case ReindexStatus.completed:
-      return 'Done!';
+      return (
+        <FormattedMessage
+          id="xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.reindexButton.doneLabel"
+          defaultMessage="Done!"
+        />
+      );
     case ReindexStatus.paused:
-      return 'Resume';
+      return (
+        <FormattedMessage
+          id="xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.reindexButton.resumeLabel"
```

Comment:
nit: it seems that some of these id's are based on the message, while others are based on the current index status.

For example:

For `ReindexStatus.inProgress` the ID is `xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.reindexButton.inProgressLabel`

Where with `ReindexStatus.completed` we have `xpack.upgradeAssistant.checkupTab.reindexing.flyout.checklistStep.reindexButton.doneLabel`